models/eval/inspirational_generation.py

```diff
@@ -260,26 +266,27 @@ def resetVar(newVal):
         if lambdaD > 0:
 
             loss = -lambdaD * model.netD(noiseOut)[:, 0]
-            sumLoss += loss
+            combinedLoss[2] += loss
 
             if not randomSearch:
                 loss.sum(dim=0).backward()
 
         if nevergrad:
             for i in range(nImages):
-                optimizers[i].tell(inps[i], float(sumLoss[i]))
```

Comment:
you could have an if nevergrad == 'moo' to avoid breaking the code when optimizing hte mono objectif